Web-Editor.md

```diff
@@ -37,42 +37,57 @@ ___
 
+### Editor Context
+
+You can add contexts in the editor in addition to in-game. To do so, either click the "Add Contexts" button before adding a permission or simply fill in the `key` and `value` fields after a permission with approriate keys and vaules, such as `world` and `nether`, or `server` and `hub`.
+ 
+Note that a permission can only have one world and one server context at a time. You have to set the same permission again with the other context to make it apply on multiple worlds/servers.
+
 #### Adding a permission
 
-To add a permission, type the node you'd like to add into the "Add Permission" input box and press enter. You can repeat this for all permissions that you want to add with the same values.  
-When typing will the editor show a list of permissions matching your input. You can select one through the arrow keys or with your cursor and select it by pressing enter or clicking on it.
+To add a permission, type the node you'd like to add into the "Add Permission" input box and press enter. You can repeat this for all permissions that you want to add with the same properties.  
+When typing the editor will show a list of suggestions matching your input so far. You can select one through the arrow keys or with your cursor and include it by pressing enter or clicking on it.
+After you've entered the permission(s), click the "+" button to add.
 
-After you've set the permission(s), go to the "+" button and press it.
+You can add multiple permissions at once! Just type or copy/paste them into the box, and add them as usual.
 
-![](https://thumbs.gfycat.com/ChubbyWarmAmmonite-size_restricted.gif)
+![](https://i.imgur.com/vyCVt7j.gif)
 
 #### Editing existing data
 
 To edit any value in the table, just click it.
 
-![](https://thumbs.gfycat.com/MiniatureIdleAxolotl-size_restricted.gif)
+![](https://i.imgur.com/s3G5MvP.gif)
 
-#### Adding Context
+#### Sorting the data
 
-To add context to a permission, either click the "Add Contexts" button before adding a permission, or click the area right under the "Contexts" column to edit the context of a permission.  
-Use "world" or "server" as keys to set per-world and per-server permissions respectively. Note that a permission can only have one world and one server context at a time. You have to set the same permission again with the other context to make it apply on multiple worlds/servers.
+To change the sorting settings, click on the column heading you'd like to sort by.
 
-![](https://thumbs.gfycat.com/SkeletalHarmoniousFattaileddunnart-size_restricted.gif)
+![](https://i.imgur.com/yVN4Lzp.gif)
 
-#### Sorting the data
+#### Luckperms Nodes
 
-To change the sorting settings, click on the column heading you'd like to sort by.
+The editor allows you to add or change aspects abouts groups and players such as weight and parents using nodes.
+This is all of them, and what they do:
```

Comment:
Bit late, but wouldn't it sound better as `This is a list of all nodes and what they do:`
That grammar is really weird to understand.

Reply:
The current wording is fine imo.

Reply:
If you say so...